src/main/java/com/Model/User.java

```diff
@@ -0,0 +1,58 @@
+package com.Model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    private String emailAddress;
+    private String password;
```

Comment:
isn't it dangerous to store the password in the clear? I'd look into how SpringSecurity does this and not reinvent the wheel if possible.